app/_src/mesh/installation/ecs.md

```diff
@@ -104,11 +104,120 @@ Services are bootstrapped with a `Dataplane` specification.
 
 Transparent proxy is not supported on ECS, so the `Dataplane` resource for a
 service must enumerate all other mesh services this service contacts and include them
-[in the `Dataplane` specification as `outbounds`][dpp-spec].
+[in the `Dataplane` specification as `outbounds`][dpp-spec]. {% if_version gte:2.11.x %}
+Since `2.11.x`, we have introduced a new feature that leverages Route53 to simplify migration to the mesh. Please see the [Dynamic Outbounds](#dynamic-outbounds) section for more details.
+{% endif_version %}
 
 See the example repository to learn
 [how to handle the `Dataplane` template with Cloudformation](https://github.com/Kong/kong-mesh-ecs/blob/main/deploy/counter-demo/demo-app.yaml#L31-L46).
 
+{% if_version gte:2.11.x %}
+#### Dynamic outbounds
+
+> [!WARNING]
+> IPv6 is not supported by this feature.
+
+In `2.11.x`, we introduced an option to leverage Route53 to simplify migration to the mesh. This functionality create Route53 domains that resolve to local addresses by DNS, which can then be used by applications. The `{{site.mesh_product_name}} Control Plane` is responsible for generating domains and local addresses, and ensures their availability to the application. It removes the burden of manually maintaining outbounds and enables faster, more automated migration.
+
+> [!NOTE]
+> AWS enforces a [limit of 5 requests per second to the Route53 API per AWS profile](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/DNSLimitations.html#:~:text=For%20the%20Amazon%20Route%2053,a%20value%20of%20Rate%20exceeded%20.). {{site.mesh_product_name}} performs initial requests for each Hosted Zone on startup, and thereafter makes additional requests only if changes are needed, at intervals of 10 seconds by default (this can be adjusted using the `KMESH_RUNTIME_AWS_ROUTE53_REFRESH_INTERVAL` setting)
```

Comment:
should we add this equation that tries to estimate the max propagation time given this 5 rps limit?